src/azurelinux/3.0/net9.0/cross/riscv64/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-crossdeps-builder-net9.0-local AS builder
+ARG ROOTFS_DIR=/crossrootfs/riscv64
+
+# Install the latest debootstrap
+RUN tdnf remove -y debootstrap && \
+    git clone --depth 1 "https://git.launchpad.net/ubuntu/+source/debootstrap" /debootstrap && \
```

Comment:
from what I can see noble was added in debootstrap 1.0.134 so I'd prefer getting the .tar.gz from https://ftp.debian.org/debian/pool/main/d/debootstrap/ instead of git

Reply:
also would be good to file an issue tracking this so we can remove it once azurelinux is updated

Reply:
It is a shell script, not an executable. We will need to setup a fakechroot etc. to make it work from the tarball, see https://src.fedoraproject.org/rpms/debootstrap/blob/rawhide/f/debootstrap.spec. BTW, these steps are taken from the readme of lanchpad  official package, to run without installing. Lets keep this for now. I will remove it once newer version of azlinux is published in a few months.

Reply:
I looked into this a bit more and it seems the only relevant change is that `scripts/noble` is a symlink to `scripts/gutsy`. I think we can just create that symlink?

Reply:
This step took 8s (out of 145mins)

```
2024-06-16T18:06:48.9108010Z #7 [builder 2/4] RUN tdnf remove -y debootstrap &&     git clone --depth 1 "https://git.launchpad.net/ubuntu/+source/debootstrap" /debootstrap &&     chmod a+x /debootstrap/debootstrap &&     ln -s /debootstrap/debootstrap -t /usr/local/bin
2024-06-16T18:06:49.0377831Z #7 0.284 Loaded plugin: tdnfrepogpgcheck
2024-06-16T18:06:50.5655548Z #7 1.812 
2024-06-16T18:06:50.5655730Z #7 1.812 Removing:
2024-06-16T18:06:50.5656173Z #7 1.812 debootstrap      noarch       1.0.128+nmu2-1.azl3 @System      193.98k     0.00b
2024-06-16T18:06:50.5658780Z #7 1.812 
2024-06-16T18:06:50.5659290Z #7 1.812 Total installed size: 193.98k
2024-06-16T18:06:50.5659535Z #7 1.812 Total download size:   0.00b
2024-06-16T18:06:50.6800990Z #7 1.927 Testing transaction
2024-06-16T18:06:50.8734176Z #7 1.930 Running transaction
2024-06-16T18:06:50.8734709Z #7 1.974 Removing: debootstrap-1.0.128+nmu2-1.azl3.noarch
2024-06-16T18:06:50.8735050Z #7 2.120 Cloning into '/debootstrap'...
2024-06-16T18:06:56.8048205Z #7 DONE 8.1s
```

riscv64 is a community-supported platform, I think we can live with it for a while. Unless you are feeling too strongly about it, I'd say lets keep it for now and we can delete it once tdnf picks up the latest version in a few weeks or so. (we have been using and trusting launchpad aka Canonical packages forever, so there is no security risk)

LMK 😅

Reply:
@am11 it's not about the time, I'm more worried about cloning directly from git HEAD. We don't really know when Azure Linux will bump to new debootstrap and if a simple `RUN ln -s gutsy /usr/share/debootstrap/scripts/noble` works too I prefer that :)

In fact I tried that and it seems to work for me locally.

Reply:
Lets merge this to unblock others and then follow-up on it. I don't wana run another 4 hours CI job for it. It hasn't changed in years and won't gonna break it. Fair?

Reply:
Lets merge this to unblock others and then follow-up on it. I don't wana run another 4 hours CI job for it. That git-clone step hasn't changed in years and hasn't broken anyone. It can sustain a few days. Fair?

Reply:
I agree, sounds good.

Reply:
Upstream PR https://github.com/microsoft/azurelinux/pull/9422